style.css

```diff
@@ -0,0 +1,107 @@
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 20px;
+}
+
+.border {
+  text-align: center;
+  border: solid 1px black;
+  padding: 10px;
+  width: 20%;
+}
+
+.list-container {
+  display: flex;
```

Comment:
you should have a utility class .flex with display flex. It would save you 6 lines of code.